certified-connectors/ascentcloudterritoryplanner/apiDefinition.swagger.json

```diff
@@ -0,0 +1,235 @@
+        "propertyName": "Categories",
+        "propertyValue": "Sales and CRM"
+    }
+],
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/territories_for_territory_plan": {
+      "get": {
+        "responses": {
+          "400": {
+            "description": "An Error Occurred"
+          },
+          "401": {
+            "description": "Unauthorized"
+          },
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "data": {
+                  "type": "object",
+                  "properties": {
+                    "id": {
+                      "type": "string",
+                      "description": "id"
+                    },
+                    "name": {
+                      "type": "string",
+                      "description": "name"
+                    },
+                    "status": {
+                      "type": "string",
+                      "description": "status"
+                    },
+                    "mapObject": {
+                      "type": "string",
+                      "description": "mapObject"
+                    },
+                    "territories": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "id": {
+                            "type": "string",
+                            "description": "id"
+                          },
+                          "name": {
+                            "type": "string",
+                            "description": "name"
+                          },
+                          "color": {
+                            "type": "string",
+                            "description": "color"
+                          },
+                          "modelSetId": {
+                            "type": "string",
+                            "description": "modelSetId"
+                          },
+                          "shapeIdAssignments": {
+                            "type": "array",
+                            "items": {
+                              "type": "string"
+                            },
+                            "description": "shapeIdAssignments"
+                          },
+                          "assigneeId": {
+                            "type": "string",
+                            "description": "assigneeId"
+                          }
+                        }
+                      },
+                      "description": "territories"
+                    }
+                  },
+                  "description": "data"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Territory Plan",
+        "description": "Get Territory Plan",
+        "operationId": "public_v1_get_territory_plan",
```

Comment:
@simongroom all operationIds should be in Camel case. please change to "PublicV1GetTerritoryPlan". Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).